ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapperTest.java

```diff
@@ -185,4 +188,21 @@ public void mapTheirDetailsFromCCDClaimWithResponse() {
             finalClaim.isMoreTimeRequested(), ccdDefendant.getResponseMoreTimeNeededOption().toBoolean());
     }
 
+    @Test
+    public void mapCCJFromToCCDDefendant() {
+        //Given
+        TheirDetails theirDetails = SampleTheirDetails.builder().organisationDetails();
+        Claim claimWithCCJ = SampleClaim.getClaimFullDefenceStatesPaidWithAcceptation();
```

Comment:
Here as discussed, don't populate the defendant response. It should be claim with default CCJ, and not defendant response.

Reply:
done